include/cartesian_geom/point.h

```diff
@@ -0,0 +1,105 @@
+#ifndef POINT_H
+#define POINT_H
+
+
+template <class Cr>
+class point
+{
+public:
+    typedef typename Cr::RT 	K;
```

Comment:
K confusing with Kernel. This is a number type so why not NT?

Reply:
FT used